README.md

```diff
@@ -1,6 +1,9 @@
-==========================
-LoadDataToSnowflake Action Plugin
-==========================
+# LoadDataToSnowflake Action Plugin
+
+[![cm-available](https://cdap-users.herokuapp.com/assets/cm-available.svg)](https://docs.cask.co/cdap/current/en/integrations/cask-market.html)
```

Comment:
should this be `https://docs.cdap.io/cdap/current/en/integrations/cask-market.html`?

Reply:
I am just merging existing commits back to develop, not making any changes like package renaming. I can fix this in the next PR.